backend/database_handler/migration/versions/37196a51038e_appeals.py

```diff
@@ -24,10 +24,14 @@ def upgrade() -> None:
     op.create_unique_constraint(
         "rollup_transactions_hash", "rollup_transactions", ["transaction_hash"]
     )
-    op.add_column("transactions", sa.Column("appealed", sa.Boolean(), nullable=False))
+    op.add_column("transactions", sa.Column("appealed", sa.Boolean(), nullable=True))
     op.add_column(
         "transactions", sa.Column("timestamp_accepted", sa.BigInteger(), nullable=True)
     )
+    op.add_column(
+        "transactions",
+        sa.Column("ghost_contract_address", sa.String(length=255), nullable=True),
+    )
     # ### end Alembic commands ###
```

Comment:
@mpaya5 could you set all values to `False` and then set it as `nullable=False`? I think we have some examples of that in other migrations

Reply:
sorry, my comment was referring to the `is appealed` column

Reply:
Okey, added in the next commit: https://github.com/yeagerai/genlayer-studio/pull/680/commits/99a14bf8e95ffd5abb9f9b9064b46f326ed55895